x-pack/plugins/triggers_actions_ui/public/application/components/simple_connector_form.tsx

```diff
@@ -92,7 +97,7 @@ const FormRow: React.FC<FormRowProps> = ({
           {!isPasswordField ? (
             <UseField
               path={id}
-              config={getFieldConfig({ label, isUrlField })}
+              config={getFieldConfig({ label, isUrlField, defaultValue })}
```

Comment:
The default value must be placed within this `config` prop otherwise it will take precedence over the actual user defined value if it was changed.